pkgs/development/tools/parsing/nex/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoPackage, fetchFromGitHub, lib }:
+let
+  rev = "1a3320dab988372f8910ccc838a6a7a45c8980ff";
+in
+buildGoPackage rec {
+  pname = "nex";
+  version = "0.0.0-20210330102341-${builtins.substring 0 12 rev}";
```

Comment:
Please use the unstable version format. We do not need the commit hash in version.